osclib/pkglistgen_comments.py

```diff
@@ -309,8 +310,11 @@ def check_staging_accept(self, project: str, target: str):
         if not approver:
             return
         sections = self.parse_sections(comment['comment'])
+        devel_target = devel_project_get(self.apiurl, target, '000package-groups')
+        if devel_target[0] is None or devel_target[1] is None:
```

Comment:
while I love mystical arrays, I think the usual way to use this function is `project, package = devel_project_get`

Reply:
I know I said that, but this was (obviously to me) pseudo code